packages/authentication/src/__tests__/unit/providers/authentication.provider.unit.ts

```diff
@@ -110,7 +110,10 @@ describe('AuthenticateActionProvider', () => {
       strategy.setMockUser(mockUser);
       provider = new AuthenticateActionProvider(
         () => Promise.resolve(strategy),
-        u => (currentUser = u),
+        (...u) => {
```

Comment:
I don't see how is this change related to the enhanced setter. Could you please explain the reasoning, or revert this part back?

Reply:
It was a left-over and should be reverted now.